examples/alanine_chain.cc

```diff
@@ -66,7 +66,7 @@ int main() {
   // manager.update("alanine-X.json");
   auto pair_manager{rascal::make_adapted_manager<rascal::AdaptorNeighbourList>(
       manager, cutoff)};
-  std::string filename{"alanine-X.json"};
+  std::string filename{"reference_data/alanine-X.json"};
```

Comment:
The file was moved to `examples/data`, but this refers to `reference_data`. I understand why (the file being referenced in the binary is the one in CMAKE_BINARY_DIR), but should the path be unified?

Reply:
The compiled example `alanine_chain` resides in the build folder. I changed the file copy from the examples to the `reference_data` folder _within_ the build folder. So when you run the compiled example, it reads from `librascal/build/reference_data/` and the example structures do not globber up the `build/` folder, which I found weird now that I saw it. All examples (and if tests compile) test files should now be in `build/reference_data/`

Reply:
yes, I understand that. What I mean is that it can be weird for a reader to see the code mentioning `reference_data/alanine-X.json`, and not finding it in `librascal/examples`. 

To fix this we could either change `librascal/tests/reference_data` to `librascal/tests/data` (and thus `librascal/build/reference_data` to `librascal/build/data`), or `librascal/examples/data` to `librascal/examples/reference_data`.

Reply:
Ok, I agree. I did just that.